scss/civicrm/contact/pages/_memberships.scss

```diff
@@ -423,6 +423,12 @@
       width: 105%;
     }
 
+    &.no-border {
+      margin-left: -20px;
+      margin-right: -20px;
+      width: 105%;
```

Comment:
105%? would this break if the screen shrinks or expands? Shouldn't it be something like `width: calc(100% - 40px);` ?

Reply:
Nope: Other classes in pages are fixed like this. so, it's not breaking. see few lines above for the same.